siggen/tests/test_siglists.py

```diff
@@ -2,11 +2,17 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import importlib
+
 import mock
 from pkg_resources import resource_stream
 import pytest
 
-from siggen import siglists_utils
+# NOTE(willkg): We do this so that we can extract signature generation into its
+# own namespace as an external library. This allows the tests to run if it's in
+# "siggen" or "socorro.signature".
+base_module = '.'.join(__name__.split('.')[:-2])
+siglists_utils = importlib.import_module(base_module + '.siglists_utils')
```

Comment:
We could use relative imports here, but we're also patching which is easier to do if we have the base module as a string so I figured I'd just do this.